attributes.go

```diff
@@ -0,0 +1,51 @@
+package bascule
+
+import (
+	"time"
+
+	"github.com/xmidt-org/arrange"
+)
+
+var nilTime = time.Time{}
+
+type BasicAttributes map[string]interface{}
+
+func (a BasicAttributes) Get(key string) (interface{}, bool) {
+	v, ok := a[key]
+	return v, ok
+}
+
```

Comment:
Is `BasicAttributes` supposed to implement `Attributes`? If so, just this one method isn't enough.  Looks like `Attributes` has several methods?

Reply:
Nope, `Attributes` just has one method.